fusillade/utils/authorize.py

```diff
@@ -56,11 +71,16 @@ def evaluate_policy(
         PaginationConfig={
             'MaxItems': 20,
             'PageSize': 8
-        })
+        }
+    )
+    if resource_policy:
+        params.update(
+            ResourcePolicy=resource_policy,
+            CallerArn='arn:aws:iam::634134578715:user/anyone')  # TODO this should be an AWS fusillade service account
```

Comment:
Is this CallerArn for the default group? Could/Should it be?

Reply:
I'm still figuring out how this field should be used. For not it's not going to be used, but it required for AWS to simulate resource policies.